Properties/launchSettings.json

```diff
@@ -25,7 +25,8 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
         "ACCOUNT_ID": "YOUR_ACCOUNT_ID",
-        "API_KEY": "YOUR_API_KEY"
+        "API_KEY": "YOUR_API_KEY",
+        "FREECLIMB_NUMBER": "YOUR_FREECLIMB_NUMBER"
```

Comment:
where is this env var used?

Reply:
I don't think it's used @katebrune. I was just going through the instructions in the ReadMe. That has since been updated in the latest push